src/models/VariableCreate.ts

```diff
@@ -0,0 +1,6 @@
+export type VariableCreate = {
+  name: string,
+} & Partial<{
```

Comment:
I personally find this hard to read when using `Partial<{...}>` when declaring types. Usually partial is used for modifying an existing type. Could we just use the `?` for these types?

Reply:
Ah yeah I forgot you're not fond of that - I'll update haha

Reply:
For the record I like it because at a glance you're able to separate the required model from the optional model though point taken about partial usage, you're absolutely right

Reply:
I'd rather just sort required at the top and optional at the bottom. I think it is more parseable to see a property and see it's optional rather than find what section it is in. But I won't die on this hill lol.

Reply:
That's an interesting idea, I think we already have (had) a class sorting rule for that - I got the model for this slightly wrong anyway but I updated this and the `VariableEdit` model so we're g2g 👍🏻

Reply:
... except I didn't push those changes before I merged 🤦🏻  I'll add them to my next PR shortly, sorry